Dockerfile

```diff
@@ -7,5 +7,9 @@ RUN apk add --update musl-dev openssl-dev openssl-libs-static \
     cargo build -p typst-cli --release
 
 FROM alpine:latest
-WORKDIR /root/
+MAINTAINER Typst <hello@typst.app>
 COPY --from=build  /app/target/release/typst /bin
+WORKDIR /home/typst
+RUN adduser -D typst
+USER typst
```

Comment:
The entrypoint change is useful, but setting the user is not. It is only a hindrance to usefulness not a security improvement. Access to the container with root permissions is still possible anyway, and now you've hindered a workflow where the calling envirenment can easily set the user ID to match the UID/GID of the user on the host system so that any volumes mounted inside the container (for example a project directory) can be read/written as the same user that owns the files instead of some arbitrary UID from the container that won't even be readable by the user that wrote them.

Reply:
Let me double check that later in the day, I left computer now :S

Reply:
I removed the user creation since accessing the root container was still possible indeed. let me know if it's better.

Reply:
It's hard to review from mobile but from what I can see that looks better. Not sure on Alpine default paths but if they prefer system package stuff in /usr and 3rd party source built stuff in /usr/local like many distros do and if the latter is in the default path you should consider putting the bin there.

Reply:
Mmh I don't think moving it to `/usr/local/bin/` or `/usr/bin/` is going to make a difference. Is it really needed?

Reply:
Needed? No of course not it works either way, but in general the closer you follow distribution expectations the less surprises there will be down the road for end users.

Reply:
Sure, but in this case, it follows FHS and I don't think moving it to one or the other place will make a difference.